src/uproot/behaviors/TBranch.py

```diff
@@ -3237,7 +3237,7 @@ def __setitem__(self, where, what):
     def __delitem__(self, where):
         del self.dict[where]
 
-    def __iter__(self, where):
+    def __iter__(self):
```

Comment:
Pretty sure this was a bug, that wasn't a valid `__iter__` and `where` wasn't used.

Reply:
That is definitely right: it must have been copy-pasted from `__getitem__`.